src/global/pidfile.cc

```diff
@@ -171,8 +171,14 @@ int pidfh::open(const ConfigProxy& conf)
   };
   int r = ::fcntl(pf_fd, F_SETLK, &l);
   if (r < 0) {
-    derr << __func__ << ": failed to lock pidfile "
-	 << pf_path << " because another process locked it." << dendl;
+    if ( errno == EAGAIN || errno == EACCES ) {
```

Comment:
i think `cpp_strerror()` would suffice here, hence no need to differentiate `EAGAIN`, `EACCESS` from other errno.

Reply:
@tchaikov 
The EACCES and EAGAIN `cpp_strerr` are not very clear by themselves, and in this explicitly indicate a lock-conflict which is the essential part of this routine. So that is why I separate them out. The other errors are probably less frequent, and mostly due to config errors and/or running on a non-locking fs.

Reply:
fair enough. could you remove the spaces before `)` and after `(` ?

Reply:
@tchaikov done.